content/systems.md

```diff
@@ -110,25 +111,26 @@ Source files submitted to the Judges will be compiled using the following comman
 - For C/C++:  the executable file generated by the compiler will be executed to generate the output of the submission.
 - For Python 3: the main source file will be executed by the PyPy3 Python3 interpreter to generate the output of the submission.
 - For Java: the compiled main class will be executed using the following command:
-  - java `-java -Dfile.encoding=UTF-8 -XX:+UseSerialGC -Xss65536k -Xms1572864k -Xmx1572864k $@`
-  - Available as `myjava`
+    - `java -Dfile.encoding=UTF-8 -XX:+UseSerialGC -Xss65536k -Xms1966080k -Xmx1966080k $@`
+    - Available as `myjava`
 - For Kotlin: the compiled main class will be executed using the following command:
-  - java `kotlin -Dfile.encoding=UTF-8 -J-XX:+UseSerialGC -J-Xss65536k -J-Xms1572864k -J-Xmx1572864k $@`
-  - Available as `mykotlin`
+    - `kotlin -Dfile.encoding=UTF-8 -J-XX:+UseSerialGC -J-Xss65536k -J-Xms1966080k -J-Xmx1966080k $@`
+    - Available as `mykotlin`
 
 Compilation and execution as described above will take place in a “sandbox” on a dedicated judging machine.
-The judging machine will be as identical as possible to, and at least as powerful as, the machines used by teams.
+The judging machine will be identical to the machines used by teams.
```

Comment:
Just double-checking: will they be identical? I assume we'll set up judgehosts in DW again, not in the cloud, but you never know :stuck_out_tongue:

Reply:
Yep that is the plan 👍